woocommerce/woocommerce
# Migrate `@woocommerce/notices` to TS

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33098 .

This PR migrates @woocommerce/notices to TS

### How to test the changes in this Pull Request:

1. Run `pnpm nx build @woocommerce/notices`
2. Observe that `packages/js/notices/build-types` is generated

### Other information:




### FOR PR REVIEWER ONLY:
